packages/cluster/src/commands/list.js

```diff
@@ -36,7 +36,9 @@ const formatStatus = status => {
 
 const run = async function (argv) {
     const clusters = await listClusters(argv)
-
+    const anyCustomContext = clusters.some(
+        cluster => cluster.contextPath !== ''
+    )
```

Comment:
Should we just always show the context, even if none are custom?

Reply:
We could. The idea was that this is most likely not relevant to most users. I think very few are actually using this context-path, and it might be confusing if you don't know what it does?

Reply:
I could go either way here... I also think it would be confusing if `d2 cluster list` sometimes renders in a different format if you happen to have a custom context on one cluster you're running.  It would make any kind of parsing of the output difficult as well, though we're not exactly well setup for that as it is